BREAKING CHANGE: use a specific xud-docker branch to launch

reliveyy opened the pull request:
This PR enables the binary launcher to use a specific xud-docker branch to setup the whole environment. It's related to https://github.com/ExchangeUnion/xud-docker/pull/836. It will download xud-docker launcher binary file into `~/.xud-docker/launcher/versions/<commit>` and delegate all CLI commands to xud-docker launcher. This is the secret behind launching different xud-docker branches with xud-launcher.

### Common Usage

#### Launch mainnet with latest xud-docker release version

```
./xud-launcher setup
```

#### Launch a feature branch of xud-docker

```
export BRANCH=awesome-feature
./xud-launcher setup
```

#### Launch simnet of xud-docker

```
export NETWORK=simnet
./xud-launcher setup
```

#### Launch older version of xud-docker

```
export BRANCH=older-version
./xud-launcher setup
```

#### Use a custom service image

```
./xud-launcher --xud.image=xud:latest
```

P.S.
1. On Windows CMD please use `set VAR=VALUE` to set environment variable before using the launcher

reliveyy committed:
BREAKING CHANGE: use a specific xud-docker branch to launch

This commit enables the binary launcher to use a specific xud-docker
branch to setup the whole environment. It's related to
ExchangeUnion/xud-docker#836. It will download xud-docker launcher
binary file into ~/.xud-docker/launcher/versions/<commit> and delegate
all CLI commands to xud-docker launcher.

raladev commented:
- [ ] cant detect python on windows env when i have it 
```
D:\xud-launcher>xud-launcher.exe setup
exec: "python3": executable file not found in %PATH%

D:\xud-launcher>python3
Python 3.9.1 (tags/v3.9.1:1e5d33e, Dec  7 2020, 17:08:21) [MSC v.1927 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> exit
Use exit() or Ctrl-Z plus Return to exit
>>> exit()
```
- [x] does not work on both my ubuntu envs. (status code 2). Python 3.6.9 may be the reason (that is installed in ubuntu by default), but both envs has 3.8 python that can be runned using `python38`, `python3.8`,`python3`

First:
```
native@ubuntu:~/xud-launcher$ sudo ./xud-launcher setup
2020/12/28 08:14:04 Found python3: /usr/bin/python3
2020/12/28 08:14:04 Found venv
2020/12/28 08:14:04 Python version: Python 3.6.9
branch=master, network=mainnet, networkDir=/home/native/.xud-docker/mainnet
exit status 2
native@ubuntu:~/xud-launcher$ BRANCH=mp NETWORK=simnet ./xud-launcher setup
2020/12/28 08:14:15 Found python3: /usr/bin/python3
2020/12/28 08:14:15 Found venv
2020/12/28 08:14:15 Python version: Python 3.6.9
branch=mp, network=simnet, networkDir=/home/native/.xud-docker/simnet
download xud-docker source zip
open 083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10.zip: permission denied
native@ubuntu:~/xud-launcher$ BRANCH=mp NETWORK=simnet sudo ./xud-launcher setup
2020/12/28 08:14:25 Found python3: /usr/bin/python3
2020/12/28 08:14:25 Found venv
2020/12/28 08:14:25 Python version: Python 3.6.9
branch=master, network=mainnet, networkDir=/home/native/.xud-docker/mainnet
exit status 2
native@ubuntu:~/xud-launcher$ sudo BRANCH=mp NETWORK=simnet ./xud-launcher setup
2020/12/28 08:16:44 Found python3: /usr/bin/python3
2020/12/28 08:16:44 Found venv
2020/12/28 08:16:44 Python version: Python 3.6.9
branch=mp, network=simnet, networkDir=/home/native/.xud-docker/simnet
download xud-docker source zip
extracting zip file
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.editorconfig
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.gitattributes
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/ISSUE_TEMPLATE/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/ISSUE_TEMPLATE/bug_report.md
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/ISSUE_TEMPLATE/feature_request.md
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/workflows/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/workflows/auto-build.yml
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.github/workflows/manual-build.yml
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/.gitignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/LICENSE
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/README.md
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/arby/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/bitcoind/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/bitcoind/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/bitcoind/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/bitcoind/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/boltz/wrapper.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/connext/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/mainnet-peers.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/rinkeby-peers.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/geth/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/litecoind/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/litecoind/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/litecoind/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/litecoind/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/litecoind/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/patches/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/patches/apply.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/patches/lnd.patch
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/patches/neutrino.patch
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc-simnet/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/lnd.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/start_tor.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/torrc
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndbtc/wait-file.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/patches/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/patches/apply.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/patches/lnd.patch
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/patches/neutrino.patch
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc-simnet/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/lnd.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/start_tor.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/torrc
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/lndltc/wait-file.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/proxy/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/proxy/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/proxy/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/proxy/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/__main__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/auto_unlock.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/check_wallets.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/close_other_utils.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/config.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/errors.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/loader.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/mainnet.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/simnet.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/template.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/testnet.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/config/xud-docker.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/core.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/errors.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/models.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/docker/registry.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/errors.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/DockerTemplate.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/arby.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/base.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/bitcoind.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/boltz.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/btcd.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/connext.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/geth.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/image.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/lnd.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/proxy.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/webui.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/node/xud.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/banner.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/command.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/history.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/shell/shell.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/table/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/table/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/table/base.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/table/color.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/table/service_table.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/types.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/utils.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/launcher/warm_up.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/requirements.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/setup.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/utils/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/Dockerfile.aarch64
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/webui/supervisord.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/.dockerignore
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/Dockerfile
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/Dockerfile.aarch64
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/entrypoint.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/update-backup-dir.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/images/xud/xud-backup.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/requirements-dev.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/setup.sh
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/banner.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/cli.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/mainnet.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/status.txt
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/test_light.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/test_simnet.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/testnet.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/utils.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/integration/xud-docker.conf
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tests/test_sample.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/build
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/build.bat
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/__init__.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/docker.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/git.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/github.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/image.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/src.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/toolkit.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/travis.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/core/utils.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/helper.py
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/push
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/push.bat
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/release
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/release.bat
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/test
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/tools/test.bat
xud-docker-083fbc2f10e6d25ee23d50b0dd2c42352e5a0f10/xud.sh
exit status 2
native@ubuntu:~/xud-launcher$ 
```
Second:
```
roma@roma-ThinkPad-X1-Carbon-6th:~$ cd xud-launcher/
roma@roma-ThinkPad-X1-Carbon-6th:~/xud-launcher$ BRANCH=mp NETWORK=simnet ./xud-launcher setup
2020/12/28 18:59:48 Found python3: /usr/bin/python3
2020/12/28 18:59:48 Python pip version: pip 19.1.1 from /home/roma/.local/lib/python3.6/site-packages/pip (python 3.6)
2020/12/28 18:59:48 Found venv
2020/12/28 18:59:48 Python version: Python 3.6.9
branch=mp, network=simnet, networkDir=/home/roma/.xud-docker/simnet
exit status 2
roma@roma-ThinkPad-X1-Carbon-6th:~/xud-launcher$ BRANCH=mp NETWORK=testnet ./xud-launcher setup
2020/12/28 19:01:41 Found python3: /usr/bin/python3
2020/12/28 19:01:42 Python pip version: pip 19.1.1 from /home/roma/.local/lib/python3.6/site-packages/pip (python 3.6)
2020/12/28 19:01:42 Found venv
2020/12/28 19:01:42 Python version: Python 3.6.9
branch=mp, network=testnet, networkDir=/home/roma/.xud-docker/testnet
exit status 2
roma@roma-ThinkPad-X1-Carbon-6th:~/xud-launcher$ BRANCH=mp ./xud-launcher setup
2020/12/28 19:01:50 Found python3: /usr/bin/python3
2020/12/28 19:01:51 Python pip version: pip 19.1.1 from /home/roma/.local/lib/python3.6/site-packages/pip (python 3.6)
2020/12/28 19:01:51 Found venv
2020/12/28 19:01:51 Python version: Python 3.6.9
branch=mp, network=mainnet, networkDir=/home/roma/.xud-docker/mainnet
exit status 2
roma@roma-ThinkPad-X1-Carbon-6th:~/xud-launcher$ ./xud-launcher setup
2020/12/28 19:01:59 Found python3: /usr/bin/python3
2020/12/28 19:01:59 Python pip version: pip 19.1.1 from /home/roma/.local/lib/python3.6/site-packages/pip (python 3.6)
2020/12/28 19:01:59 Found venv
2020/12/28 19:01:59 Python version: Python 3.6.9
branch=master, network=mainnet, networkDir=/home/roma/.xud-docker/mainnet
download xud-docker source zip
extracting zip file
xud-docker-e427bac8a32054849bf8e89367988933862c102d/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.editorconfig
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.gitattributes
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/ISSUE_TEMPLATE/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/ISSUE_TEMPLATE/bug_report.md
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/ISSUE_TEMPLATE/feature_request.md
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/workflows/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/workflows/auto-build.yml
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.github/workflows/manual-build.yml
xud-docker-e427bac8a32054849bf8e89367988933862c102d/.gitignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/LICENSE
xud-docker-e427bac8a32054849bf8e89367988933862c102d/README.md
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/arby/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/bitcoind/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/bitcoind/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/bitcoind/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/bitcoind/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/boltz/wrapper.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/connext/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/mainnet-peers.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/rinkeby-peers.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/geth/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/litecoind/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/litecoind/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/litecoind/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/litecoind/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/litecoind/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/patches/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/patches/apply.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/patches/lnd.patch
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/patches/neutrino.patch
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc-simnet/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/lnd.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/start_tor.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/torrc
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndbtc/wait-file.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/patches/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/patches/apply.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/patches/lnd.patch
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/patches/neutrino.patch
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc-simnet/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/lnd.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/start_tor.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/torrc
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/lndltc/wait-file.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/proxy/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/proxy/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/proxy/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/proxy/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/__main__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/auto_unlock.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/check_wallets.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/close_other_utils.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/config.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/errors.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/loader.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/mainnet.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/simnet.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/template.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/testnet.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/config/xud-docker.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/core.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/errors.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/models.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/docker/registry.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/errors.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/DockerTemplate.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/arby.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/base.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/bitcoind.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/boltz.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/btcd.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/connext.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/geth.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/image.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/lnd.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/proxy.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/webui.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/node/xud.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/banner.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/command.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/history.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/shell/shell.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/table/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/table/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/table/base.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/table/color.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/table/service_table.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/types.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/utils.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/launcher/warm_up.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/requirements.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/setup.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/utils/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/Dockerfile.aarch64
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/webui/supervisord.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/.dockerignore
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/Dockerfile
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/Dockerfile.aarch64
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/entrypoint.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/update-backup-dir.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/images/xud/xud-backup.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/requirements-dev.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/setup.sh
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/banner.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/cli.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/mainnet.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/status.txt
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/test_light.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/test_simnet.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/testnet.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/utils.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/integration/xud-docker.conf
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tests/test_sample.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/build
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/build.bat
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/__init__.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/docker.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/git.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/github.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/image.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/src.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/toolkit.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/travis.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/core/utils.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/helper.py
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/push
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/push.bat
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/release
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/release.bat
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/test
xud-docker-e427bac8a32054849bf8e89367988933862c102d/tools/test.bat
xud-docker-e427bac8a32054849bf8e89367988933862c102d/xud.sh
exit status 2
```


Master works better:
```
native@ubuntu:~/xud-launcher$ BRANCH=mp NETWORK=simnet sudo ./xud-launcher setup
./xud-launcher gen
Exported /home/native/.xud-docker/mainnet/docker-compose.yml
Exported /home/native/.xud-docker/mainnet/data/config.json

./xud-launcher pull
Pulling lndbtc  ... done
Pulling lndltc  ... done
Pulling connext ... done
Pulling xud     ... done
Pulling boltz   ... done
Pulling proxy   ... done

./xud-launcher up -- -d proxy
Creating mainnet_proxy_1 ... done

./xud-launcher up -- -d lndbtc lndltc connext
Creating mainnet_lndltc_1  ... done
Creating mainnet_lndbtc_1  ... done
Creating mainnet_connext_1 ... done

```

raladev reviewed (changes requested):
Above

reliveyy commented:
> cant detect python on windows env when i have it

The python runtime is no longer needed in xud-launcher now. @raladev

reliveyy committed:
fix

- toml config key
- commit dit empty
- don't show "ERROR: exit status 1" in last run

reliveyy committed:
fix

- open xud-docker.conf
- mkdir ~/.xud-docker/launcher
